connectors/src/api/create_connector.ts

```diff
@@ -53,17 +68,42 @@ const _createConnectorAPIHandler = async (
       });
     }
 
-    const connectorCreator =
-      CREATE_CONNECTOR_BY_TYPE[req.params.connector_provider];
+    let connectorRes: Result<string, Error>;
+    switch (req.body.type) {
+      case "oauth": {
```

Comment:
Shouldn't this be a function of the connector provider rather than a request param ? 🤔